packages/react-scripts/bin/react-scripts.js

```diff
@@ -8,9 +9,14 @@ case 'build':
 case 'eject':
 case 'start':
 case 'test':
+  var scriptArgs = [require.resolve('../scripts/' + script)].concat(args);
+  var debugFlag = getDebugFlag(args);
+  if (debugFlag) {
+    scriptArgs.unshift(debugFlag);
```

Comment:
note, running `node --debug-brk ./node_modules/.bin/react-scripts test --debug-brk` will biff on conflicting port access to 5858, if this wasn't clear already.  I did not handle this case, but it may be worth discussion?